introduction.md

```diff
@@ -1,4 +1,8 @@
 Introduction for myself to dojo
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?